package.json

```diff
@@ -125,5 +125,6 @@
       "rollup@>=4.0.0 <4.22.4": ">=4.22.4",
       "fuels": "0.96.1"
     }
-  }
```

Comment:
We're going to use ASDF for controlling pnpm versions in the near future. So for now setting the verison in  `package.json`'s  engine should suffice